wiki/Beatmap_Spotlights/id.md

```diff
@@ -0,0 +1,202 @@
+2. Peserta hanya dapat memiliki satu skor peringkat mingguan per Playlist.
+   - Memutar ulang sebuah playlist pada minggu yang lain hanya akan memperbarui skor terbaik dan skor baru yang lebih baik ini akan menimpa skor lama yang lebih buruk.
+3. Peserta tidak akan diberi tahu tentang peringkat musiman mereka. Sebagai gantinya, mereka akan dimasukan kedalam braket liga.
+   - Hanya peringkat 50 teratas di Leaderboard musiman yang peringkat tepatnya akan terlihat.
+   - Braket ditetapkan berdasarkan peringkat dalam Leaderboard. Peserta akan dikelompokan kedalam braket yang sesuai dengan peringkat mereka.
+4. Braket yang ditetapkan akan diumumkan setelah minggu ke dua pada 1 musim.
+
+### Hadiah
+
+*Sebagian dari hadiah ini masih menunggu implementasi. Mereka akan ditambahkan seiring berjalannya musim. Maka dari itu, hal ini dapat berubah sewaktu-waktu.*
+
+Hadiah akan dibagikan kepada timeshift lobby mingguan, mapper dari beatmap yang dikurasi, dan setiap peserta yang mendapat peringkat di Leaderboard musiman.
+
+1. 10 teratas dari setiap timeshift lobby mingguan akan diberikan hadiah osu! supporter tag selama 1 minggu.
+2. Selama musim berlangsung, setiap peserta akan ditempatkan ke braket, yang ditunjukkan oleh Lencana Profil Unik yang terdapat di profil mereka selama musim berlangsung.
+   - Lencana akan diperbarui setiap minggu setelah minggu ke dua pada 1 musim.
+   - Pemain terbaik selama 1 musim mungkin saja dapat mempertahankan lencana tersebut secara permanen. Namun, rincian tentang hal tersebut masih belum diputuskan.
+3. Di setiap akhir musim, mapper terbaik, sesuai keputusan curator, akan mendapatkan hadiah supporter tag selama 1 bulan.
+4. Di setiap akhir musim, medali baru yang tidak bisa dibuka akan ditambahkan. Pemain harus menyelesaikan setiap beatmap musim ini setidaknya satu kali untuk mendapatkannya.
+
+| Lencana | Bracket Tier | Peringkat |
+| :-: | :-- | :-- |
+| ![](img/badges/autumn_2020/osu/ri_2.png) ![](img/badges/autumn_2020/osu/ri_3.png) | Rhythm Incarnate | Top 50 |
+| ![](img/badges/autumn_2020/osu/diamond_2.png) ![](img/badges/autumn_2020/osu/diamond_3.png) | Diamond | Top 51 – 3% |
+| ![](img/badges/autumn_2020/osu/platinum_2.png) ![](img/badges/autumn_2020/osu/platinum_3.png) | Platinum | 3% – 10% |
+| ![](img/badges/autumn_2020/osu/gold_2.png) ![](img/badges/autumn_2020/osu/gold_3.png) | Gold | 10% – 25% |
+| ![](img/badges/autumn_2020/osu/silver_2.png) ![](img/badges/autumn_2020/osu/silver_3.png) | Silver | 25% – 50% |
+| ![](img/badges/autumn_2020/osu/bronze_2.png) ![](img/badges/autumn_2020/osu/bronze_3.png) | Bronze | 50% – 70% |
+| ![](img/badges/autumn_2020/osu/copper_2.png) ![](img/badges/autumn_2020/osu/copper_3.png) | Copper | 70% – 95% |
+| ![](img/badges/autumn_2020/osu/iron_2.png) ![](img/badges/autumn_2020/osu/iron_3.png) | Iron | 95% – 100% |
+
+Tabel ini hanya menunjukkan satu dari empat versi Lencana. Setiap mode permainan memiliki versi lencana nya sendiri.
+
+### Sistem Kurasi
```

Comment:
`Kurasi` shouldn't be capitalized here to be consistent with how the other headings are being written